arch/arm64/src/imx9/Kconfig

```diff
@@ -302,6 +302,10 @@ config IMX9_FLEXSPI
 	bool "ENABLE FLEXSPI interface"
```

Comment:
Rename to "Enable FLEXSPI Interface"

Rational: NuttX used Capitalized by default:

$ git grep "ENABLE " | grep bool
arch/arm64/src/imx9/Kconfig:	bool "ENABLE FLEXSPI interface"

$ git grep "Enable " | grep bool | wc
   1077    7208   72654